server/mockapi/router.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import express from 'express';
+import { UIServerModule } from 'types';
+
+const moduleName = 'mockapi';
+
+export const MockApiModule: UIServerModule = {
+  moduleName,
+  addModule: (logGenerator) => {
```

Comment:
Same comment re log generator, but maybe as @ajborley progresses it can be revisited